Makefile

```diff
@@ -19,6 +19,7 @@ pc-dangerous.img: ubuntu-core-desktop-22-amd64-dangerous.model $(EXTRA_SNAPS)
 	tar czSf $@ $<
 
 pc.img.xz: pc.img
+	rm -f pc.img.xz
 	xz --threads=0 -vv $<
```

Comment:
Could use `xz --force` to do the overwrite.

Note also instead of putting `pc.img.gz` it would be better to use `$@`.

Reply:
Done.